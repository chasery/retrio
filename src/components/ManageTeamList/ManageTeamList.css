.ManageTeamList {
  margin-top: 1rem;
  box-shadow: 0 0.0625rem 0.0625rem #0000001c;
  background-color: #fff;
}
.ManageTeamList > li {
  display: flex;
  flex-direction: column;
  list-style: none;
}
.ManageTeamList > li + li {
  border-top: 0.0625rem solid #ebebeb;
}
.ManageTeamList > li > * {
  padding: 1rem;
}
