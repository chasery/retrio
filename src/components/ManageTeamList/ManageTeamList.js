import React from 'react';
import ManageTeamItem from '../ManageTeamItem/ManageTeamItem';
import './ManageTeamList.css';

function ManageTeamList(props) {
  const { members } = props;

  const renderTeamMembers = (members) => {
    if (members) {
      return members.map((member) => (
        <ManageTeamItem
          key={member.id}
          id={member.id}
          name={member.name}
          email={member.email}
          owner={member.owner}
        />
      ));
    }
  };

  return <ul className='ManageTeamList'>{renderTeamMembers(members)}</ul>;
}

export default ManageTeamList;
