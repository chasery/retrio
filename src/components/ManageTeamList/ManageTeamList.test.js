import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ManageTeamList from './ManageTeamList';

describe('ManageTeamList component', () => {
  it('renders without error', () => {
    const wrapper = shallow(<ManageTeamList />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
