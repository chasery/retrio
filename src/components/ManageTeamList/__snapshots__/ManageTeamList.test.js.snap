// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ManageTeamList component renders without error 1`] = `
<ul
  className="ManageTeamList"
/>
`;
